config/merge.go

```diff
@@ -326,3 +316,13 @@ func asString(obj interface{}) string {
 	}
 	return ""
 }
+
+// IsValidRemote checks if the specified string is a valid remote (for builds)
+func IsValidRemote(remote string) bool {
+	switch {
+	case urlutil.IsGitURL(remote), urlutil.IsURL(remote):
```

Comment:
@thaJeztah Lol, because… I did have more case at first :angel:. But you're absolutely right :wink: